graylog2-server/src/main/java/org/graylog2/indexer/results/IndexQueryResult.java

```diff
@@ -45,19 +32,10 @@ public String getOriginalQuery() {
     }
 
     public String getBuiltQuery() {
-        try {
-            return XContentHelper.convertToJson(builtQuery, false);
-        } catch (IOException ignored) {
-            // exception comes from InputStream, but that stream is from a byte array, so won't do IO.
-        }
-        return null;
-    }
-
-    public TimeValue took() {
-        return took;
+        return builtQuery;
     }
 
-    static List<ResultMessage> buildResults(SearchHits hits) {
-        return StreamSupport.stream(hits.spliterator(), false).map(ResultMessage::parseFromSource).collect(Collectors.toList());
+    public long tookMs() {
```

Comment:
We need to consider that this is breaking consumers, i.e. the map widget plugin.